lib/pbench/server/api/__init__.py

```diff
@@ -272,21 +274,48 @@ class Upload(Resource):
     def post(self):
         global app
         if not request.headers.get("filename"):
-            app.logger.debug("Missed filename in header")
+            app.logger.debug("Tarfile upload: Missed filename in header")
```

Comment:
Oh, you drew attention to these; I'd skimmed over them. "Missed filename" sounds awkward here. Something more like "Tarfile upload POST operation requires a `filename` header to name the uploaded file." ???